sapi/fpm/php-fpm.conf.in

```diff
@@ -125,6 +125,17 @@
 ; Default value: 10
 ;systemd_interval = 10
 
+; Bind the master process to a cpu set.
+; The value can be one cpu id, a range or a list thereof.
+;
+; Default Value: not set
+; the master process could be bound to a single core
+; process.cpu_list = "4"
+; a range from the minimum cpuid to the max required
+; process.cpu_list = "0-3"
+; or multiple ranges separated by a comma
+; process.cpu_list = "1-4,8-16,32-48"
```

Comment:
It would be better to have the format docs before the value rather than actual example. See other entries to get some inspiration. For example https://github.com/php/php-src/blob/64b8306fd8c2252d31481f43eaf1e5874f2bd5b0/sapi/fpm/www.conf.in#L276-L283 .